src/idaplugin/defs.cpp

```diff
@@ -20,7 +20,8 @@ int runCommand(
 		const std::string& cmd,
 		const std::string& args,
 		intptr_t* pid,
-		bool showWarnings)
+		bool showWarnings,
+		void** hdl)
```

Comment:
Is it possible that you forgot to commit the corresponding change in function declaration in `defs.h`? I'm unable to compile this, because `defs.h` does not have `void** hdl` argument. It works when I do this in `defs.h` (note the default value, it is important for the next comment):
```c
int runCommand(
		const std::string& cmd,
		const std::string& args,
		intptr_t* pid = nullptr,
		bool showWarnings = false,
		void** hdl = nullptr);
```
What is your code? Do you have something different in there?

Reply:
Yes it is exactly right, I forgot to update the header definition.  Thanks for finding commit it now.